install/src/Help.hs

```diff
@@ -0,0 +1,149 @@
+-- |Module for Help messages and traget descriptions
+module Help where
+
+import           Development.Shake
+import           Data.List                                ( intersperse
+                                                          , intercalate
+                                                          )
+
+import Env
+import Print
```

Comment:
Maybe format the imports in the same style?

Reply:
Done